lib/control.ml

```diff
@@ -150,28 +130,32 @@ implicit module Option = struct
     | None -> acc
     | Some x -> f x acc
 
-  let traverse (type a) (type b) {F : Applicative} (f : a -> b F.t)  : a option -> b t F.t = function
+  (* Traversable *)
+  let traverse (type a) (type b) {F : Applicative} (f : a -> b F.t)  : a option -> b option F.t = function
     | None -> F.return None
     | Some x -> F.fmap (fun x -> Some x) (f x)
 end
 
-implicit module List = struct
+implicit module List : sig
```

Comment:
Should there be a similar thing for the option type?

Reply:
Yep I've already done that, just not committed yet